ui/pages/send/send.utils.js

```diff
@@ -36,6 +44,30 @@ function isBalanceSufficient({
   return balanceNumeric.greaterThanOrEqualTo(totalAmount);
 }
 
+function gasIsExcessive({
```

Comment:
With this method only being used in the send duck, it should probably exist in the ducks/send/helpers file instead of in the page utils.

Reply:
Okay I will move it

Reply:
Should this now be deleted?

Reply:
Now it is, i missed it